adagrams/game.py

```diff
@@ -1,11 +1,110 @@
+from random import randint
+from copy import deepcopy
+
+LETTER_POOL = [
+                "A", "A", "A", "A", "A", "A", "A", "A", "A",
+                "B", "B",
+                "C", "C",
+                "D", "D", "D", "D", 
+                "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", 
+                "F", "F",
+                "G", "G", "G",
+                "H", "H",
+                "I", "I", "I", "I", "I", "I", "I", "I", "I", 
+                "J",
+                "K",
+                "L", "L", "L", "L", 
+                "M", "M",
+                "N", "N", "N", "N", "N", "N", 
+                "O", "O", "O", "O", "O", "O", "O", "O", 
+                "P", "P",
+                "Q",
+                "R", "R", "R", "R", "R", "R", 
+                "S", "S", "S", "S", 
+                "T", "T", "T", "T", "T", "T", 
+                "U", "U", "U", "U", 
+                "V", "V",
+                "W", "W",
+                "X",
+                "Y", "Y",
+                "Z"
+    ]
+
+
 def draw_letters():
-    pass
+    # create a copy of letter pool so we don't alter original list
+    letter_pool_copy = deepcopy(LETTER_POOL)
+    drawn_letters = []
+    
+    # until the length of drawn_letters is 10, keep generating
+    # a random index to pop off from the letter pool 
+    # and append popped element to drawn_letters
+    while len(drawn_letters) < 10:
+        rand_index = randint(0, len(letter_pool_copy)-1)
+        drawn_letters.append(letter_pool_copy.pop(rand_index))
+    return drawn_letters
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    # create a copy of letter bank so we don't alter original list
+    letter_bank_copy = deepcopy(letter_bank)
+    word = word.upper()
+    valid_word = True
```

Comment:
This is soooo pythonic! 🐍 I adore how y'all set a boolean for `valid_word` at the top of the function and reassign on L56 depending on the condition, then return this boolean only once in L60! While the following is acceptable:
```
if [valid conditional]:
    return True
else:
    return False
```
What y'all did (set a boolean variable to have only one return statement at the end of the function, like below) is a Pythonista's dream! Great work!
```
valid = True
if [not valid conditional]:
    valid = False
return valid
```